src/appshell/qml/MuseScore/AppShell/welcomedialogmodel.cpp

```
@@ -30,22 +30,21 @@ using namespace mu::appshell;
-    item1.insert("title", muse::qtrc("appshell/welcome", "Enjoy free cloud storage"));
-    item1.insert("imageUrl", "qrc:/resources/welcomedialog/MuseScoreCom.png");
+    item1.insert("title", muse::qtrc("appshell/welcome", "What’s new in MuseScore Studio"));
+    item1.insert("imageUrl", "qrc:/resources/welcomedialog/WhatsNew.png");
     item1.insert("description", muse::qtrc("appshell/welcome",
-                                           "Save your scores privately on MuseScore.com to revisit past versions and invite others to view and comment – and when you’re ready, share your music with the world."));
-    item1.insert("buttonText", muse::qtrc("appshell/welcome", "View my scores online"));
-    item1.insert("destinationUrl",
-                 "https://musescore.com/my-scores?utm_source=mss-app-welcome-musescore-com&utm_medium=mss-app-welcome-musescore-com&utm_campaign=mss-app-welcome-musescore-com");
+                                           "Includes essential new engraving tools, major improvements to playback, video export, dive notation for guitar, and features to speed up your workflow."));
+    item1.insert("buttonText", muse::qtrc("appshell/welcome", "Watch video"));
+    item1.insert("destinationUrl", "https://youtu.be/grKX-cBEEmM");
```

Cppcheck (2.20.0)

[error] There is an unknown macro here somewhere. Configuration is required. If Q_PROPERTY is a macro then please configure it.

(unknownMacro)